src/comments.js

```diff
@@ -834,7 +834,33 @@ function printComment(commentPath, options) {
       return "#" + comment.value;
     case "CommentBlock":
     case "Block":
-      return "/*" + comment.value + "*/";
+      const lines = comment.value.split("\n");
+      const isJsDocComment =
+        lines.length > 1 &&
+        lines.slice(0, lines.length - 1).every(line => line.trim()[0] === "*");
+      if (!isJsDocComment) {
+        return "/*" + comment.value + "*/";
+      }
+
+      // Determine the indentation level of the parent node.
+      const parentNode = commentPath.getParentNode();
+      const indentLevel = parentNode.loc.start.column;
```

Comment:
Here we're trying to use the parent node of the comment to determine the indent level. This works when the parent is already properly indented, but if prettier needs to re-indent the parent, then the comment's indentation level will be where the parent was before it got re-indented.